debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresSchema.java

```diff
@@ -159,4 +221,8 @@ protected static TableId parse(String table) {
     public TypeRegistry getTypeRegistry() {
         return typeRegistry;
     }
+
+    public List<String> getToastableColumnsForTableId(TableId tableId) {
+        return tableIdToToastableColumns.getOrDefault(tableId, Collections.unmodifiableList(new ArrayList<>()));
```

Comment:
`Collections.emptyList()` might be better in this case

Reply:
Done.